changelog_unreleased/typescript/pr-7631.md

```diff
@@ -0,0 +1,9 @@
+#### TypeScript 3.8 ([#7631](https://github.com/prettier/prettier/pull/7631) by [@thorn0](https://github.com/thorn0))
+
+Prettier doesn't yet fully support the new syntax added in TypeScript 3.8:
+
+| Syntax                                                                                                                          | `typescript` parser | `babel-ts` parser |
+| ------------------------------------------------------------------------------------------------------------------------------- | ------------------- | ----------------- |
+| [Type-Only Imports and Exports](https://devblogs.microsoft.com/typescript/announcing-typescript-3-8/#type-only-imports-exports) | ✔️                  | ❌                |
+| [ECMAScript Private Fields](https://devblogs.microsoft.com/typescript/announcing-typescript-3-8/#ecmascript-private-fields)     | ✔️                  | ✔️                |
+| [`export * as ns`](https://devblogs.microsoft.com/typescript/announcing-typescript-3-8/#export-star-as-namespace-syntax)        | ❌                  | ❌                |
```

Comment:
`export * as ns` seems to be supported with `babel-ts`.
[Playground](https://deploy-preview-7631--prettier.netlify.com/playground/#N4Igxg9gdgLgprEAuEcAeAHCAnGACAKjwEMBnPAMwgkuwgFs8ByKiJgbhABoQIMYAltFLJQxbHQDuABXEIRKYgBtJxAJ4ieAI2zEwAazgwAyhj0CoAc2QxsAVzg8AFjHpKA6k4HxSZsHGN5bwEAN281ZHBSTRALUjhcaV1LemJkCmV4ngArUjQAIV0DI2NiejgAGQs4dMzHEFy0YwtLJTgARTsIeFqlLJAzbHjsSK1iLTglAFoYGIxsCxh3AQATGCdkAA4ABh55iHj3XQxI+bhhkJqeAEcu+CS+BRAyKag4OBWP7hBsOFuBX5JYgpNJIDJ9erxegCGz2SEtNqdbo1MF1HgwcbLNYbJAAJnRugEShaAGEGKlIucAKzfOzxAAq4wU4P6IQcAEkoJ9YMYwAt+ABBLnGGBqNq9eIAX0lQA
)
```
parser=babel-ts
```
**Input:**
```ts
export * as foo from './foo';
```
**Output**
```ts
export * as foo from "./foo";
```

Reply:
You're right. Overlooked this somehow. I'll fix the changelog.